src/VirtoCommerce.NotificationsModule.Core/Types/VerificationEmailNotification.cs

```diff
@@ -0,0 +1,22 @@
+using VirtoCommerce.NotificationsModule.Core.Model;
+
+namespace VirtoCommerce.NotificationsModule.Core.Types
+{
+    public class VerificationEmailNotification : EmailNotification
+    {
+        public VerificationEmailNotification() : base(nameof(VerificationEmailNotification))
+        {
+            //for backward compatibility v.2
+            Alias = "EmailVerificationNotification";
```

Comment:
does new feature have/need any backward compatibility? Maybe it's not needed?

Reply:
Removed